spec/models/person_spec.rb

```diff
@@ -41,14 +41,16 @@
 require 'rails_helper'
 
 describe Person, type: :model do
+  let(:person) { create(:person, :with_member_access, :with_password) }
+  let(:log) { create(:project_log) }
 
+  before { person.project_logs << log }
```

Comment:
You should be able to use `let(:log) { build(:project_log) }` (hopefully?) before you create the person, and then in the person, do `create(:person, :with_member_access, :with_password, project_logs: [log])` and avoid this `before` call (they work really funky and i make an effort to avoid them when possible.